spec/helpers/interruption_screen_helper_spec.rb

```diff
@@ -249,7 +239,7 @@
 
     it "returns a list of questions affected by the soft validation" do
       expect(soft_validation_affected_questions(question, lettings_log).count).to eq(2)
-      expect(soft_validation_affected_questions(question, lettings_log).map(&:id)).to eq(%w[age1 ecstat1])
+      expect(soft_validation_affected_questions(question, lettings_log).map(&:id)).to eq(%w[ecstat1 age1])
```

Comment:
is there a reason that this changed order, and if the order is not important and it's just a touch flakey, you could
```suggestion
      expect(soft_validation_affected_questions(question, lettings_log).map(&:id)).to match_array %w[ecstat1 age1]
```

Reply:
Yup, that's a good idea